config/config_helpers.go

```diff
@@ -500,7 +500,7 @@ func getModulePathFromSourceUrl(sourceUrl string) (string, error) {
 
 	return matches[1], nil
 }
-
+var SOPSCACHE = make(map[string]string)
```

Comment:
A few NITs to  fix:

1. Put newlines around this var declaration.
1. Use the variable name `sopsCache`. That  (a) keeps the variable private to this package and (b) is more inline with conventions we use in Go.
1. Add a comment above the variable  explaining why we have a cache and what the keys and values are here.